lib/gui/app/os/windows-network-drives.js

```diff
@@ -107,6 +107,8 @@ const getWindowsNetworkDrives = async () => {
       // eslint-disable-next-line no-magic-numbers
       return [ str.slice(0, colonPosition + 1), _.trim(str.slice(colonPosition + 1)) ]
     })
+    // eslint-disable-next-line no-magic-numbers
+    .filter((couple) => couple[1].length > 0)
```

Comment:
I'd like to see a note somewhere why we're doing this. I mean, we know now, but in a couple of days ... This VMware Tools case seems like a special one worth documenting.

Reply:
imho in a comment block over that line

Reply:
This is not vmwere specific.
The second part is supposed to be `\\<ip address>\folder`.
If it's an empty string we don't use it.